prog/detect/sensors-detect

```diff
@@ -3121,7 +3121,7 @@ sub initialize_cpu_list
 sub print_cpu_info
 {
 	my $cpu = $cpu[0];
-	print "# Processor: $cpu->{'model name'} ($cpu->{'cpu family'}/$cpu->{model}/$cpu->{stepping})\n";
+	print "# Processor: $cpu->{'model name'} ($cpu->{'cpu family'}/$cpu->{model}/$cpu->{stepping}/$cpu->{cpu}/$cpu->{revision})\n";
```

Comment:
How about the following? Printing 'model name', 'cpu family', etc., on ppc doesn't really make sense.
```suggestion
	if ($kernel_arch =~ m/^ppc(64(le)?)?$/) {
		print "# Processor: $cpu->{cpu} ($cpu->{revision})\n";
	} else {
		print "# Processor: $cpu->{'model name'} ($cpu->{'cpu family'}/$cpu->{model}/$cpu->{stepping})\n";
	}
```

Reply:
I implemented something similar in c28d20d19d620.